osbenchmark/worker_coordinator/runner.py

```diff
@@ -2358,3 +2377,116 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
 
     def __repr__(self, *args, **kwargs):
         return "retryable %s" % repr(self.delegate)
+
+class DeleteMlModel(Runner):
+    @time_func
+    async def __call__(self, opensearch, params):
+        body= {
+            "query": {
+                "match_phrase": {
+                    "name": {
+                        "query": params.get('model-name')
+                    }
+                }
+            },
+            "size": 1000
+        }
+
+        model_ids = set()
+        try:
+            resp = await opensearch.transport.perform_request('POST', '/_plugins/_ml/models/_search', body=body)
+            for item in resp['hits']['hits']:
+                doc = item.get('_source')
+                if doc:
+                    id = doc.get('model_id')
+                    if id:
+                        model_ids.add(id)
+        except:
+            pass # no current model
+
+        for model_id in model_ids:
+            resp=await opensearch.transport.perform_request('POST', '/_plugins/_ml/models/' + model_id + '/_undeploy')
+            resp=await opensearch.transport.perform_request('DELETE', '/_plugins/_ml/models/' + model_id)
+
+    def __repr__(self, *args, **kwargs):
+        return "delete-ml-model"
+
+class RegisterMlModel(Runner):
+    @time_func
+    async def __call__(self, opensearch, params):
+        config_file = params.get('model-config-file')
+        if config_file:
+            with open(config_file, 'r') as f:
+                body = json.loads(f.read())
+        else:
+            body = {
+                "name": params.get('model-name'),
+                "version": params.get('model-version'),
+                "model_format": params.get('model-format')
+            }
+        search_body = {
+            "query": {
+                "match": {
+                    "name": body['name']
+                }
+            },
+            "size": 1000
```

Comment:
Will we ever need to change the size parameter? If so, might be useful to have a variable that references its value